tests/integration/raw_studies_blueprint/test_download_matrices.py

```diff
@@ -101,6 +101,9 @@ def update_general_data(self, internal_study_id: str, **data: t.Any):
         res.raise_for_status()
 
 
+from tests.integration.prepare_proxy import PreparerProxy
```

Comment:
it's very confusing to import in the middle of the file a class that has the same name as the class defined in this file: is it on purpose ?
There must be a better way !

Reply:
I agree, but I think that Laurent, wanted to avoid repetition in code. I will see.